packages/react-components/src/PasswordStrength.tsx

```diff
@@ -1,52 +1,78 @@
 // Copyright 2017-2020 @polkadot/react-components authors & contributors
 // SPDX-License-Identifier: Apache-2.0
 
+import { useTranslation } from '@polkadot/react-components/translate';
```

Comment:
`import { useTranslation } from  './translate'` - this is local to the component (so also needs to move down in position)